### ISSUE TITLE
Toolbar icon should be under More

### ISSUE THREAD

**Issue by twMat** [danielo515/TW5-EncryptTiddlerPlugin#7]
The lock icon should be more discretely placed inside the dropdown *More* tool.

While perhaps easy to access when it's fully visible, it is also visually disturbing and even confusing for plublic TSs - and it is of course for public TWs that the whole plugin is particularly relevant.

The button, i.e to encrypt, is probably  also not a "daily action" like the buttons Edit or Close.

Ideally, one would be able to set it's display under Controlpanel>Appearance>Toolbar

**Comment by danielo515** [comment]
Hello @twMat 

This should be a user's decission. Since the wiki that exposes the plugin is to demonstrate the plugin's usage, I think I will keep it as default. Otherwise, the button is not properly configured, and that's why it will appear as default wherever you install the plugin. This is because the plugin is older than TW's buttons mechanism, and I did not updated it. I will add that to the list!

**Comment by twMat** [comment]
>This should be a user's decission.

Yes, exactly my point. To have it visible by default is fine, but there is no obvious way to move it from there. If you know of one then a "better than nothing" could be to include it in *readme*.

**Comment by mandarvaze** [comment]
I know this is REALLY old issue.

as @twMat suggested - I only have "Close" and "More" visible by default. Everything else is hidden under "More"

The Lock icon is indeed visible there, but

Clicking on it does not seem to work. i.e. I do not see floating password edit box and encrypt button.

The floating password edit box and encrypt button are visible only when the lock icon is visible next to "X" button.

Is the "widget" expected to popup when invoked via "more" menu ?

**Comment by twMat** [comment]
@mandarvaze , AFAICT I should close this issue.  What you bring up is appears to be a totally different issue but OK, let's see if we can solve it before I close this.

You say that the Lock icon is visible under the *tiddler* "More" button... but is that really so?

On tiddlywiki.com , the Lock button can be found under the wiki sidebar *Tools* tab, where it is called "Set password" which gives this:

![image](https://user-images.githubusercontent.com/4534056/65376816-e10d4000-dca4-11e9-9d67-96fb1a0a4834.png)

Is your wiki different?

**Comment by mandarvaze** [comment]
@twMat  Ha ha. Your original message (now deleted) did confuse me 😄

**Comment by danielo515** [comment]
The problem is that, the buttons under more are not expected to open a in-place popup. When you click the encrypt button, the popup opens, but because the "main" popup closes you don't see it.
If you open the more button again, you will see it open: 

![image](https://user-images.githubusercontent.com/2270425/104048851-39db5180-51e4-11eb-9d39-821cbf0c0be7.png)

A possible solution will be to trigger a modal in the middle of the screen, but that will make less obvious which tiddler you are encrypting.
